### PR TITLE
Adds burrito

### PR DESCRIPTION
**Pull request by xXBenjaminYoungXx**
Listed the best food,
Added comment on 'Next-time'

### CODE REVIEW

**Review by KipHamiltons — COMMENTED** [review]
The PR title should reflect the changes made.
Otherwise looks good to me (LGTM) once you merge `main` into this branch, which is probably worth doing, because you'll have to deal with a merge conflict (fun stuff!!)

**Review by hflyn — DISMISSED** [review]
Last name corrected

**Review by BryceTuppurainen — APPROVED** [review]
Merge conflicts were resolved by Harry

**Pull request merged**
